framework/wazuh/core/cluster/master.py

```diff
@@ -227,6 +256,42 @@ def to_dict(self):
                            'last_keep_alive': self.last_keepalive}
                 }
 
+    def _cmd_syn_w_g_e(self, data: bytes):
+        logger = self.task_loggers['Agent-groups send']
+        start_time = self.send_agent_groups_status['date_start']
+        return c_common.end_sending_agent_information(logger, start_time, data.decode())
+
+    def _cmd_syn_wgc_e(self, data: bytes):
+        logger = self.task_loggers['Agent-groups send full']
+        start_time = self.send_agent_groups_status['date_start']
+        return c_common.end_sending_agent_information(logger, start_time, data.decode())
+
+    def _cmd_syn_w_g_err(self, data: bytes):
+        logger = self.task_loggers['Agent-groups send']
+        return c_common.error_receiving_agent_information(logger, data.decode(), info_type='agent-groups')
+
+    def _cmd_syn_wgc_err(self, data: bytes):
+        logger = self.task_loggers['Agent-groups send full']
+        return c_common.error_receiving_agent_information(logger, data.decode(), info_type='agent-groups')
+
+    def _cmd_dapi(self, data: bytes):
+        self.server.dapi.add_request(self.name.encode() + b'*' + data)
+        return b'ok', b'Added request to API requests queue'
+
+    def _cmd_get_nodes(self, data: bytes):
+        cmd, res = self.get_nodes(json.loads(data))
+        return cmd, json.dumps(res).encode()
+
+    def _cmd_get_health(self, data: bytes):
+        cmd, res = self.get_health(json.loads(data))
+        return cmd, json.dumps(
+            res, default=lambda o: "n/a" if isinstance(o, datetime) and o == utils.get_date_from_timestamp(0) else
+            (o.__str__() if isinstance(o, datetime) else None)).encode()
+
+    def _cmd_sendsync(self, data: bytes):
+        self.server.sendsync.add_request(self.name.encode() + b'*' + data)
+        return b'ok', b'Added request to SendSync requests queue'
+
```

Comment:
Missing docstring, review this in all newly added methods.

Reply:
Added Docstrings to all new methods